PROPOSAL.md

```diff
@@ -0,0 +1,181 @@
+  Enforcing a specific encoding would harm interoperability.
+* The ability to pass account components unaltered removes the need to link complex account identifier computation libraries to each ledger client.
+* Having owners and subaccounts in the transaction log enables new kinds of applications.
+  For example, it becomes possible to build auxiliary smart contracts acting as replicators indexing the transaction log or sending out notifications (note, however, that the standard [does not specify how to access the transaction log](#unspecified-transaction-structure)).
+* Implementations storing unaltered principals and subaccounts allow clients to recover their state fully by replaying records from the transaction log.
+* Two-component structure maps well to the [Rosetta API specification of account identifiers](https://www.rosetta-api.org/docs/api_identifiers.html#account-identifier).
+
+### Namespaces
+
+All ICRC-1 [methods](https://github.com/dfinity/ICRC-1#methods) have the `icrc1_` prefix.
+This feature allows existing smart contracts to implement ICRC-1 methods without introducing name clashes.
+
+### Unspecified transaction structure
+
+ICRC-1 _does not_ specify the structure of transactions or provide methods for fetching them.
+That deliberate design choice enables backward compatibility with existing token ledgers, such as the ICP ledger.
+The Ledger & Tokenization working group plans to work on an _optional_ core extension specifying the interface for fetching transactions.
+
+
+### Metadata access
+
+ICRC-1 defines a [generic metadata mechanism](https://github.com/dfinity/ICRC-1#metadata) allowing ledger implementations to define application-specific attributes, such as token logos and URLs (see the [icrc1_metadata](https://github.com/dfinity/ICRC-1#icrc1_metadata-) method).
+
+
+### Modularity
+
+ICRC-1 is the _base_ standard designed for extension with follow-up specifications as new flows become practical.
+Many efficient flows require unique features of the IC platform, such as [Threshold ECDSA signatures](https://medium.com/dfinity/threshold-ecdsa-the-key-ingredient-behind-the-internet-computers-bitcoin-and-ethereum-cf22649b98a1), to become stable and well-understood.
+The standard defines the [icrc1_supported_standards](https://github.com/dfinity/ICRC-1#icrc1_supported_standards) method listing all standards the token ledger supports.
+This method will allow applications such as wallets and Rosetta API implementations to discover and take advantage of additional features in the future.
+
+The Ledger & Tokenization working group will continue working on a set of _core_ extensions recommended for all ledger implementations.
```

Comment:
```suggestion
The Ledger & Tokenization working group will continue working on a set of _optional_ extensions recommended for all ledger implementations.
```
The phrase: "core extensions" is one that contradicts itself by the definition of the word 'core' and of the word 'extension'. 'optional' makes it clear that the ICRC-1 standard stands on its own.

Reply:
Good point, rephrased the sentence.